Include/cpython/weakrefobject.h

```diff
@@ -59,3 +59,5 @@ Py_DEPRECATED(3.13) static inline PyObject* PyWeakref_GET_OBJECT(PyObject *ref_o
     return Py_None;
 }
 #define PyWeakref_GET_OBJECT(ref) PyWeakref_GET_OBJECT(_PyObject_CAST(ref))
+
+PyAPI_FUNC(void) _PyWeakref_ClearRef(PyWeakReference *self);
```

Comment:
I would prefer to restore it where it was in Python 3.12, before PyWeakref_GET_OBJECT().

Reply:
Done